src/transactionsFactories/smartContractTransactionsFactory.ts

```diff
@@ -34,12 +41,14 @@ export class SmartContractTransactionsFactory {
     private readonly abi?: IAbi;
     private readonly tokenComputer: TokenComputer;
     private readonly dataArgsBuilder: TokenTransfersDataBuilder;
+    private readonly argSerializer: ArgSerializer;
 
     constructor(options: { config: Config; abi?: IAbi; tokenComputer: TokenComputer }) {
         this.config = options.config;
         this.abi = options.abi;
         this.tokenComputer = options.tokenComputer;
         this.dataArgsBuilder = new TokenTransfersDataBuilder();
+        this.argSerializer = new ArgSerializer();
```

Comment:
In this file, maybe the code, as it was, was all right.

Reply:
Reverted.